bot.py

```diff
@@ -140,11 +140,11 @@ def formatGameserverMessage(server_name, server_details):
         return formatted_server_message
 
     # Starts the Discord Client connection
-    client.run(TOKEN)
+    client.run('TOKEN')
```

Comment:
This change should be reverted. TOKEN is referring to the global variable TOKEN above.